### PR TITLE
Feature/id 12/categoy event section

### PR DESCRIPTION
**Pull request by CatCoder36**
**Review:**
@Karvlox 
@gabitrujillo

### CODE REVIEW

**Review by Karvlox — DISMISSED** [review]

File: apps/CEventService.API/DTOs/Event/EventFilterDto.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+public class EventFilterDto
+{
+    public string? Category { get; set; }
+    public DateTime? StartDate { get; set; }
+    public DateTime? EndDate { get; set; }
+    public decimal? MinPrice { get; set; }
+    public decimal? MaxPrice { get; set; }
+    public string? Status { get; set; }
+    public string? SortBy { get; set; } // El campo para ordenar (por ejemplo, "EventDate")
+    public bool IsDescending { get; set; } = false; // True para orden descendente
```
Comment (JP-Arequipa) [inline apps/CEventService.API/DTOs/Event/EventFilterDto.cs]: delete comments

File: apps/cevent_frontend/src/app/page.tsx [hunk 2]
```diff
@@ -3,11 +3,21 @@
 import Layout from '@/components/Layout';
 import { EventCategory } from '@/utils/interfaces/Categories';
 import CategoryEventSection from '@/components/EventSection/CategoryEventSection';
+import { CategoryObj } from '@/utils/interfaces/Categories'; // Asegúrate de importar CategoryObj correctamente
```
Comment (JP-Arequipa) [inline apps/cevent_frontend/src/app/page.tsx]: delete comment

File: apps/cevent_frontend/src/services/EventService.ts [hunk 3]
```diff
@@ -1,25 +1,42 @@
...
+      StartDate: filter.StartDate,
+      EndDate: filter.EndDate,
+      MinPrice: filter.MinPrice,
+      MaxPrice: filter.MaxPrice,
+      Status: filter.Status,
+      SortBy: filter.SortBy,
+      IsDescending: filter.IsDescending,
+    };
+
+    // Realiza la solicitud a la API de Next.js usando Axios
```
Comment (JP-Arequipa) [inline apps/cevent_frontend/src/services/EventService.ts]: delete comments

**Comment by JP-Arequipa** [comment]
CI is failing @CatCoder36

**Review by gabitrujillo — APPROVED** [review]

**Review by JP-Arequipa — APPROVED** [review]

**Review by Karvlox — APPROVED** [review]

**Pull request merged**